<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="OMI" ignored="0" name="My Test" tests="2" failures="1" timestamp="15 Nov 2018 16:16:29 GMT" time="30.766" errors="0">
  <testcase name="loginWithValidCredentials" time="6.502" classname="jameel.banKChalo.tests.MyFirstTest">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.fail(Assert.java:53)
at jameel.banKChalo.tests.MyFirstTest.loginWithValidCredentials(MyFirstTest.java:26)
at jameel.banKChalo.testUtils.DynamicRunner.createTestRunner(DynamicRunner.java:127)
... Removed 50 stack frames]]>
    </failure>
  </testcase> <!-- loginWithValidCredentials -->
  <testcase name="loginWithValidCredentialsViral" time="6.01" classname="jameel.banKChalo.tests.MyFirstTest"/>
</testsuite> <!-- My Test -->
